interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props is short for properties. this is used as from parent to child components. meaning to pass from one component to another. Props are both used with and without destructuring. Props is for
+  
+  Researched answer: Javascript library for building user interface free and open source. Use is for state management and rendering the state of the DOM structure in mobile applications or even server rendering. React is in entities called components that are React code. this is declarative programming. Class based components, virtual DOM, lifecycle methods, JSX, React hooks are noteworthy features of React.
```

Comment:
While I like the research you came up with, try to keep the researched answer still expanding on the original question.